gcloud/logging/logger.py

```diff
@@ -298,7 +298,7 @@ def list_entries(self, projects=None, filter_=None, order_by=None,
                   more entries can be retrieved with another call (pass that
                   value as ``page_token``).
         """
-        log_filter = 'logName:%s' % (self.name,)
+        log_filter = 'logName=%s' % (self.full_name,)
```

Comment:
Ok, now I see this in fact not a bug at all on the client side or the server side.The list syntax difference was just my mistake. Without a resource type and without an equal sign to use an index, the query is just extremely slow. The same nextPageToken keeps coming back because it's indicating that the same query is still taking place, even if there are still no results. The code will work as is, but can take several minutes to get results even on a brand new project.

So yeah, we should still make this change, but it's an optimization, not a bug fix. Given how sluggish the logging API can be, we might even consider changing all the samples to polling the request until the token is None.